sirepo/pkcli/roles.py

```diff
@@ -70,13 +69,16 @@ def _parse_args(uid_or_email, roles):
 
     # POSIT: Uid's are from the base62 charset so an '@' implies an email.
     if '@' in uid_or_email:
-        u = sirepo.auth.email.AuthEmailUser.search_by(user_name=uid_or_email)
+        m = sirepo.auth.get_module('email')
+        assert m, \
+            f'uid_or_email={uid_or_email} contains "@" but email not configured'
+        u = m.user_by_user_name(uid_or_email)
     else:
-        u = sirepo.auth_db.UserRegistration.search_by(uid=uid_or_email)
+        u = sirepo.auth.registered_user(uid_or_email)
```

Comment:
I'm feeling like these should be unchecked_get_user or something like that

Reply:
968f7f1f

By "these" what do you mean? That auth.email.user_by_user_name should also be prefixed with unchecked because we don't check if they are logged in?

Reply:
Yes, because they can return a None result, when they don't do anything.

Reply:
d19b0996